lib/widgets/action_sheet.dart

```diff
@@ -104,13 +109,14 @@ Future<String?> fetchRawContentWithFeedback({
     // - If request(s) take(s) a long time, show snackbar with cancel
     //   button, like "Still working on quote-and-reply…".
     //   On final failure or success, auto-dismiss the snackbar.
+    final zulipLocalizations = ZulipLocalizations.of(context);
     try {
       fetchedMessage = await getMessageCompat(PerAccountStoreWidget.of(context).connection,
         messageId: messageId,
         applyMarkdown: false,
       );
       if (fetchedMessage == null) {
-        errorMessage = 'That message does not seem to exist.';
+        errorMessage = zulipLocalizations.errorMessageDoesNotSeemToExist;
```

Comment:
nit: These three commit messages should have a verb:

8b9ea0fb8 action_sheet: Translations
159c85421 dialog: Translations
d4b0cd2b4 lightbox: Translations

"Add translations", like in several of your other commits, would do nicely.